Jenkinsfile

```diff
@@ -81,7 +81,7 @@ pipeline {
                           echo 'Starting kserver...'
                           k-distribution/target/release/k/bin/spawn-kserver kserver.log
                           cd k-exercises/tutorial
-                          make -j`nproc` ${MAKE_EXTRA_ARGS}
+                          make -j`nproc` --output-sync ${MAKE_EXTRA_ARGS}
```

Comment:
Did you try `--output-sync=recurse` instead?

Reply:
Yes. This way is easier to read. Every call is surrounded by `enter` and `leaving` directory messages. The output is longer, but it makes it easier to track down failing tests.